<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcSouceDialog</class>
 <widget class="QDialog" name="AcSouceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AcSource设置页（仅AC版本可用）</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo.qrc">
    <normaloff>:/icon/image/3.jpeg</normaloff>:/icon/image/3.jpeg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>ID输入范围：1~24</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="id_label">
         <property name="text">
          <string>ID号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="id_comboBox"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>电压输入范围：30~280V</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="volt_label">
         <property name="text">
          <string>电压：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="volt_lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="volt_pushButton">
         <property name="text">
          <string>设置</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>频率输入范围：10~70Hz</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="fre_label">
         <property name="text">
          <string>频率：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fre_lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="fre_pushButton">
         <property name="text">
          <string>设置</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
